src/Symfony/Component/Worker/composer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "symfony/worker",
+    "type": "library",
+    "description": "Library to build workers",
+    "keywords": ["worker", "consumer", "queue", "message", "amqp"],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Grégoire Pineau",
+            "email": "lyrixx@lyrixx.info"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "http://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.5.9",
+        "ext-pcntl": "*",
```

Comment:
this makes the component incompatible with Windows (and so any project depending on it), as pcntl does not exist for Windows. This should be an optional dependency